contracts/modules/PauseGuardian/ProtocolPausableGuardian.sol

```diff
@@ -20,7 +20,28 @@ contract ProtocolPausableGuardian is State, PausableGuardian {
         _setTarget(this._isPaused.selector, target);
         _setTarget(this.toggleFunctionPause.selector, target);
         _setTarget(this.toggleFunctionUnPause.selector, target);
+        _setTarget(this.pause.selector, target);
+        _setTarget(this.unpause.selector, target);
         _setTarget(this.changeGuardian.selector, target);
         _setTarget(this.getGuardian.selector, target);
     }
+
+    function pause(bytes4 [] calldata sig)
```

Comment:
these functions should be in PausableGuardian.sol instead

Reply:
Yes, i"ll remove